jdaviz/core/marks.py

```diff
@@ -725,6 +725,12 @@ def __init__(self, viewer, overlay, **kwargs):
     def overlay(self):
         return self._overlay
 
+    def set_selected_style(self, is_selected: bool = False) -> None:
```

Comment:
We don't currently use type-hinting anywhere, so l think let's avoid introducing it now and having a mixed codebase.

```suggestion
    def set_selected_style(self, is_selected):
```

Reply:
I am saddened, but I understand. Good catch!